medical-hub/src/component/Footer/Footer.jsx

```diff
@@ -1,14 +0,0 @@
-import React from "react";
-import Footer3 from "./Footer3";
-import Footer4 from "./Footer4";
-import Footer5 from "./Footer5";
-
-export default function Footer() {
-  return (
-    <div>
-      <Footer3 />
-      <Footer4 />
-      <Footer5 />
```

Comment:
Give some meaningful names for your components. Footer3, Footer4 etc is not determining any significant definition of that component.